src/__test__/itemsCount.test.js

```diff
@@ -0,0 +1,23 @@
+import itemsCount from '../JSFunctions/itemsCount.js';
+
+describe('Test itemsCount function', () => {
+  document.body.innerHTML = `
+  <span id='items-num'></span>
+  `;
```

Comment:
> - [ ] Optional Changes

I would kindly suggest that you remove this part so that your code is in adherence to `YAGNI`principles. The reason for stating this is because you are not passing the result of the array as the innerHTML of this code block and you are not testing the length in this file.

Reply:
Thanks for your opinion, sir. I will work on that